opus/application/test_api/api_test_helper.py

```diff
@@ -166,3 +167,25 @@ def _run_csv_equal_file(self, url, exp_file):
         print('Expected:')
         print(expected)
         self.assertEqual(resp, expected)
+
+    def _run_zip_equal(self, url, expected):
+        print(url)
+        response = self._get_response(url)
+        self.assertEqual(response.status_code, 200)
+        if '__cart/download.json' in url:
```

Comment:
Rather than check explicitly for download,json and download/ how about passing in a flag that says what kind of response you expect? Then it's still useable if we change the API in the future.